### PR TITLE
new module that enables debugging for every new ua

### PR DESCRIPTION
**Pull request by simbabque**
The idea behind this is that it can be used in any program. Right now it's not possible to configure it, but I am planning to add some kind of store for created user agents, so that it's possible to retrieve them via some imported DSL. That way it would be possible to change settings for individual user agents, which might be especially useful if there are several of those in a large application.

### CODE REVIEW

File: t/LWP/ConsoleLogger/Everywhere.t [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            'code lives'
+        );
+    };
+    ok $stderr, 'there was a dump';
+}
+
+{
+
+    package Foo::Bar;
+
```
Comment (eilara) [inline t/LWP/ConsoleLogger/Everywhere.t]: Why is this required? Is this a test for the feature "works in all modules not just the one where you used it"? If so, why is the previous required?

Comment (simbabque) [inline t/LWP/ConsoleLogger/Everywhere.t]: Yes, it's a test for "works in other modules". You're right, we don't really need the first one.

**Comment by simbabque** [comment]
I'm done with this feature. If you like it please go ahead and merge it. :)

**Comment by oalders** [comment]
@simbabque this is wonderful.  Thank you!
